pimcore/models/Object/ClassDefinition/Data/Multihref.php

```diff
@@ -394,6 +400,8 @@ public function getDataForGrid($data, $object = null, $params = [])
     public function getVersionPreview($data, $object = null, $params = [])
     {
         if (is_array($data) && count($data) > 0) {
+            $pathes = [];
```

Comment:
Same as before (although here probably it doesn't change behaviour). Nevertheless I'd leave this as it was and let PHP trigger a warning - make it easier to find. A TODO couldn't hurt though :)